src/toil_scripts/lib/urls.py

```diff
@@ -8,7 +8,7 @@
 from toil_scripts.lib.programs import docker_call
 
 
-def download_url(url, work_dir='.', name=None, s3_key_path=None, cghub_key_path=None):
+def download_url(url, work_dir='.', name=None, synapse_login=None, s3_key_path=None, cghub_key_path=None):
```

Comment:
I didn't realize we decided against synapse. It was not supposed to be in this PR, so I am fine with removing it. I think it's a good option for storing reference data and would like to use it in the future.